api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -133,3 +160,31 @@ def get_census_version_directory() -> Dict[CensusVersionName, CensusVersionDescr
 
     # Cast is safe, as we have removed all aliases
     return cast(Dict[CensusVersionName, CensusVersionDescription], directory)
+
+
+def get_census_mirrors() -> CensusMirrors:
```

Comment:
if this is a public method, it needs a docstring

Reply:
I marked it as private. We want a public method to return all the mirrors, but I think we should only return the name and the geographical location, and possibly the provider. @pablo-gar any thoughts? Do we want to expose it immediately?

Reply:
Also, if a public API, naming should be consistent between this and `get_census_version_directory`.  I'd recommend `get_mirror_directory` so that we don't change the aforementioned existing API

Reply:
We do need a public API whose output produces a list of values that can be directly used as an argument for `open_soma(mirror = [VALUE])`

I recommend a function `get_mirror_names() -> List[str]`

@ebezzi can you add that please

Reply:
Will do

Reply:
I don't see how `get_mirror_names` is useful unless it *also* returns information about the mirror, sufficient for the user to pick one.  So I think:
1. it needs to return the entire mirror record, not just the name
2. we may want (over time, later) enhance the mirror record with some additional (human readable?) info that helps this process

Reply:
Agree, it needs to return the region and  the provider at minimum. I was thinking to omit the `base_uri`, but that could actually be useful if the user wants to explore the mirror via other means. I'll just return a list of the whole records.

Reply:
sounds good - and that makes it match what we do with `get_census_version_directory`.  Suggest renaming it as well, e.g., `get_mirror_directory`.  That will distinguish it from other methods we might add in the future...  (eg.., get_current_mirror?)

Reply:
I added the method.